code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -489,7 +489,8 @@
 	. = ..()
 
 	for(var/mob/living/carbon/human/future_pancake in loc)
-		run_over(future_pancake)
+		if(has_gravity(src))
+			run_over(future_pancake)
```

Comment:
Move the gravity check out of the loop

Instead of checking if we have gravity for every human in the location (which is probably always `TRUE` or always `FALSE`), check if we have gravity before determining if we should squish people

Reply:
how bout now

Reply:
Looks good, did you test it?

Reply:
yep works fine